src/NSwag.CodeGeneration.CSharp/Models/CSharpControllerTemplateModel.cs

```diff
@@ -77,6 +77,12 @@ public class CSharpControllerTemplateModel : CSharpTemplateModelBase
         /// <summary>Gets a value indicating whether to allow adding cancellation token.</summary>
         public bool UseCancellationToken => _settings.UseCancellationToken;
 
+        /// <summary>Gets a value indicating whether to allow adding model validation attributes</summary>
+        public bool UseModelValidationAttributes => _settings.UseModelValidationAttributes;
+
+        /// <summary>Gets the type of the attribte used to specify a parameter as required.</summary>
+        public string RequiredAttributeType => _settings.RequiredAttributeType;
```

Comment:
Why do we need a setting for that? Can't we just check the ControllerTarget (AspNet/AspNetCore) and "hardcode" the full attribute name? Don't see a scenario where one would choose another type...

Reply:
Implement it the same way as latest version of the AspNetNamespace property?

Reply:
done